src/Tmds.Ssh/PrivateKeyParser.RsaPkcs1.cs

```diff
@@ -0,0 +1,147 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+
+namespace Tmds.Ssh;
+
+partial class PrivateKeyParser
+{
+    /// <summary>
+    /// Parses an RSA PKCS#1 PEM formatted key. This is a legacy format used
+    /// by older ssh-keygen and openssl versions for RSA based keys.
+    /// </summary>
```

Comment:
Do you think it's important to support this older format?

Reply:
Probably less important as time goes on but it was the default format used by ssh-keygen and openssl for a long time and there’s probably still people running on older hosts. The implementation is quite simple as well so from a maintenance perspective it’s not too bad. The only reason I can see for not implementing it would be to force people to use a newer and more secure format but at that point you may ad well not support the key format altogether (even as plaintext).

Reply:
Do you have an idea when `ssh-keygen` was using this format (as the default)?

Reply:
I'm fine including this, if you think it is still relevant.

Reply:
Looks like it was OpenSSH 7.8 (2018-08-24) https://www.openssh.com/txt/release-7.8

> ssh-keygen(1): write OpenSSH format private keys by default instead of using OpenSSL's PEM format

I think most supported distros have a newer version so up to you if you still wish to include it. I think it's ok to include as it's not too complicated but there shouldn't be too many people who need it.

Reply:
I prefer to leave it out then. For things like this, I like users to create issues and upvote them to know they are important to include.

Reply:
It has been removed.